### DISCUSSION TITLE
Adding BMS Support

### DISCUSSION THREAD

**Discussion by Cyberb29** [Louisvdw/dbus-serialbattery#428] [Ideas]
Can you add support for a Freedom Won eTower and SolarMD battery BMS?

**Comment by Louisvdw** [comment]
Both these already have native Victron support.

**Reply by Cyberb29** [reply]
Yes, however the communcation method supported is Canbus, is it possible you can add the battery to your supported list to be monitored via RS485?

**Reply by mr-manuel** [reply]
If you want the BMS to be added please open a feature request under the issue section and provide the protocol details. You can request them from the manifacturer.
